go.mod

```diff
@@ -1,61 +1,65 @@
 module terraform-provider-delphix
 
-go 1.21.1
+go 1.22.0
+
 
 require (
-	github.com/delphix/dct-sdk-go/v10 v10.0.0
-	github.com/hashicorp/terraform-plugin-sdk v1.17.2
+	github.com/delphix/dct-sdk-go/v14 v14.0.0
```

Comment:
regarding the v14 DCT update requirement:

1) We should add a small support matrix to our main doc landing page.
2) The two doc links are now broken with the doc migration change a few months ago.

Reply:
Agree